.github/workflows/soos-dast-docker-image-beta.yml

```diff
@@ -22,10 +22,9 @@ jobs:
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
-      - name: Build and Push
-        uses: docker/build-push-action@v5
-        with:
-          context: .
-          push: ${{ github.event_name != 'pull_request' }}
-          tags: soosio/dast:beta
-          labels: ${{ steps.meta.outputs.labels }}
+      - name: Promote alpha to beta
+        id: promote-alpha-to-beta
+        run: |
+          docker pull soosio/dast:alpha
+          docker tag soosio/dast:alpha soosio/dast:beta
```

Comment:
```suggestion
          docker tag soosio/dast:alpha beta
```
I think the second part doesn't have the image name... at least that is what I did locally

Reply:
This is what I get for `docker tag soosio/dast:alpha beta`

<img width="1048" alt="1" src="https://github.com/soos-io/soos-dast/assets/94386102/039af9d6-31ae-4c19-be4e-8678bcae504c">

Vs `docker tag soosio/dast:alpha soosio/dast:beta`

<img width="1051" alt="2" src="https://github.com/soos-io/soos-dast/assets/94386102/f23a627e-fcfc-4fb1-bc3d-e6d899a087fb">

Reply:
👍 maybe it was different in the docker hub context